### ISSUE TITLE
Hardcoded block count cemented not total

### ISSUE THREAD

**Issue by zhyatt** [nanocurrency/nano-node#3238]
Currently using total count for this but should be using cemented going forward. Around line: https://github.com/nanocurrency/nano-node/blob/develop/ci/record_rep_weights.py#L33 which produces this file: https://github.com/nanocurrency/nano-node/blob/develop/rep_weights_live.bin

**Comment by zhyatt** [comment]
> @dsiganos One thing to consider with this change is whether any of the activities triggered by this hardcoded value are helping generate the very confirmations that are needed to reach that value. For instance, if frontier confirmation or other key activities that help the node cement blocks don't fully active until this threshold, then changing this to be a cemented count could prove problematic or less efficient, and may need some discussion.

I think I was a bit confused in the above message, so can likely be ignored.

**Comment by dsiganos** [comment]
This is what I am doing:
1) Run a node on V21.3RC2 until reasonable synced
2) Make the following change to script record_rep_weights.py
```
-block_height = max(0, int(p["count"]) - args.cutoff)
+block_height = max(0, int(p["cemented"]) - args.cutoff)
```
3) Run the script against the synced V21.3RC2 node in step 1) to create the file rep_weights_live.bin
4) Commit rep_weights_live.bin to git

**Comment by zhyatt** [comment]
Closed by #3254
